tests/metaworld/envs/mujoco/sawyer_xyz/test_scripted_policies.py

```diff
@@ -64,6 +64,8 @@
     ['pick-place-wall-v2', SawyerPickPlaceWallV2Policy(), 0.1, 0.92, {}],
     ['push-wall-v2', SawyerPushWallV2Policy(), 0.0, 0.95, {}],
     ['push-wall-v2', SawyerPushWallV2Policy(), 0.1, 0.85, {}],
+    ['stick-pull-v2', SawyerStickPullV2Policy(), .0, 0.98, {}],
+    ['stick-pull-v2', SawyerStickPullV2Policy(), .1, 0.88, {}],
 ]
```

Comment:
This `test_cases` list shouldn't be here. I think maybe it got re-added to master when someone was rebasing an older PR. The `test_cases` variable gets overwritten before pytest runs, so it's really not doing anything here.

Reply:
ok yeah was wondering why we had it. thought it might have been just for documentation or something. I'l remove it

Reply:
have you addressed this @adibellathur?